recipes/loos/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -x
+# This install script is intended for conda-forge, and assumes the conda env
+# is already set up. If you need to set up a conda env, I suggest running
+# conda_build.sh instead.
+set -x
+
+#if [ "$platform" = "Linux" ]; then
+#    export CXX=`which g++`
+#elif [ "$platform" = 'Darwin' ]; then
+#    export CXX=`which clang++`
+#else
+#    echo "Unknown platform $platform, assuming g++"
+#    export CXX=`which g++`
+#fi
+echo "Begin loos build.sh"
+which $CXX
+export CONDA_PREFIX=$BUILD_PREFIX
```

Comment:
Shouldn't this be `$PREFIX`?

Reply:
Sorry for the slow response -- I didn't get a notification.

No. PREFIX is the install location, CONDA_PREFIX is the root of the conda install. So, $CONDA_PREFIX/bin is where the binaries are, $CONDA_PREFIX/lib is where the libraries are, etc.